### DISCUSSION TITLE
Have config path be an optional parameter

### DISCUSSION THREAD

**Discussion by benwa** [blazcode/Wallpaper-Refresher#7] [Ideas]
If there are multiple configs with a different default `bgi` set, one must put all of BgAssist in separate folders. If we could call `bgassist.exe BgAssist-Config.exe.config` instead (with it defaulting to the same directory, we would only have to have one copy of the BgAssist folder.

For example, say we had a `bgi` for workstations and servers. We could have a `workstation.config` and `server.config` and pass that straight to bgassist.exe

**Comment by blazcode** [comment]
I think that this is another good idea and is worth implementing. I need to think through how to refactor this a little. I didn't put much thought into the .config implementation or have much experience here. I was going for an easy win when first writing.
